manim/mobject/opengl/opengl_mobject.py

```diff
@@ -559,23 +582,26 @@ def set_points(self, points):
         self.refresh_bounding_box()
         return self
 
-    def apply_over_attr_arrays(self, func):
+    def apply_over_attr_arrays(
+        self, func: Callable[[npt.NDArray], npt.NDArray]
```

Comment:
```suggestion
    T = TypeVar(bound=npt.NDArray)
    def apply_over_attr_arrays(
        self, func: Callable[[T], T]
```
Maybe? This way `attr` retains the same type (although it doesn't really matter because it gets set via `setattr`)

Reply:
Done!

Reply:
I opted for something similar:
```py
if TYPE_CHECKING:
    T = TypeVar("T")
# ...

class OpenGLMobject:
    # ...
    def apply_over_attr_arrays(
        self, func: Callable[[npt.NDArray[T]], npt.NDArray[T]]
    ) -> Self:
        # ...
```